packages/vats/src/proposals/network-proposal.js

```diff
@@ -57,27 +56,23 @@ export const registerNetworkProtocols = async (vats, dibcBridgeManager) => {
 };
 
 /**
- * Create the network and IBC vats; produce `networkVat` in the core / bootstrap
- * space.
+ * Create the network and IBC vats; produce `portAllocator` in the core /
+ * bootstrap space.
  *
- * The `networkVat` is CLOSELY HELD in the core space, where later, we claim
- * ports using `E(networkVat).bindPort(_path_)`. As discussed in
- * `ProtocolHandler` docs, _path_ is:
- *
- * - /ibc-port/NAME for an IBC port with a known name or,
- * - /ibc-port/ for an IBC port with a fresh name.
+ * The `portAllocator` is CLOSELY HELD in the core space, where later, we claim
+ * ports using `E(portAllocator).allocateIBCPort`, for example.
  *
  * Contracts are expected to use the services of the network and IBC vats by way
  * of such ports.
  *
  * Testing facilities include:
  *
- * - loopback ports: `E(networkVat).bindPort('/local/')`
- * - an echo port: `E(vats.network).bindPort('/ibc-port/echo')`
+ * - loopback ports: `E(portAllocator).allocateLocalPort()`
+ * - an echo port: `E(portAllocator).allocateIBCPort()`
```

Comment:
This doesn't produce `/ibc-port/echo`, so it loses the port identity that the outside world could rely on (like how ICS-20 is usually found at `/ibc-port/transfer`).